paasta_tools/utils.py

```diff
@@ -2152,6 +2152,9 @@ def get_cluster(self) -> str:
     def get_dashboard_links(self) -> Mapping[str, Mapping[str, str]]:
         return self.config_dict["dashboard_links"]
 
+    def get_cr_owners(self) -> Dict[str, str]:
+        return self.config_dict["cr_owners"]
```

Comment:
mypy says `paasta_tools/utils.py:2156: error: TypedDict "SystemPaastaConfigDict" has no key 'cr_owners'` - we'll need to add an entry for `cr_owners` to https://github.com/Yelp/paasta/blob/ee0d2bd94d8707420dc7ac18a17e78c6f8a729a8/paasta_tools/utils.py#L1885